script/package.json

```diff
@@ -10,10 +10,10 @@
     "coffeelint": "1.15.7",
     "colors": "1.1.2",
     "donna": "1.0.16",
-    "electron-chromedriver": "^9.0.0",
-    "electron-link": "0.4.1",
+    "electron-chromedriver": "^6.0.0",
```

Comment:
`electron-chromedriver@6.0.0` won't work because of a test I modified that now requires `electron-chromedriver` to be >= 9.0.0. So `script/bootstrap` fails.

Those tests are in [`script/lib/check-chromedriver-version.js`](https://github.com/atom/atom/blob/00553e6/script/lib/check-chromedriver-version.js#L29-L33)

Those tests are kind of silly though, because if we want `electron-chromedriver` and `electron-mksnapshot` to be >v9, we'll specify them like that in `package.json`. Maybe those tests should be deleted.

Reply:
I resumed working based on [your branch](https://github.com/atom/atom/blob/electron-6.1.12-merge-master/script/package.json), the `electron-chromedriver` is v9, and the build is [failing only on linux at build stage](https://github.visualstudio.com/Atom/_build/results?buildId=72394&view=logs&j=50a69d0a-7972-5459-cdae-135ee6ebe312&t=daec06e4-a147-5611-53a8-ab34aed5d6ca) when downloading electron core.

Actually I like the checks you did, it's an adapted convention to save the next person from pulling their hair out, trying to figure out what went wrong, such failures due to dependencies mismatch, doesn't always show a useful error message, specially with your code for re-downloading binaries, I would rather keep the checks.

Reply:
Alright, thanks for letting me know.

I don't understand why CI failed. It might pass if restarted? It works for me locally.